packages/website/public/schema.yml

```
@@ -9,6 +9,64 @@ servers:
 tags:
   - name: NFT Storage
 paths:
+  /store:
+    post:
+      tags:
+        - NFT Storage
+      summary: Store an ERC-1155 compatible NFT
+      description: |
+        Store an [ERC-1155](https://eips.ethereum.org/EIPS/eip-1155)-compatible NFT as 
+        a collection of content-addressed objects connected by IPFS CID links.
+
+        The POST request accepts `multipart/form-data` content, which must contain a form field named `meta`.
+
+        The `meta` field must contain a JSON string that conforms to the [ERC-1155 metadata schema](https://eips.ethereum.org/EIPS/eip-1155#metadata).
+
+        Any field inside the `meta` object whose value is set to `null` will be treated as a file reference, 
```

I think this is bit misleading,  and I would phrase it more like this

```suggestion
        Any field(s) inside the `meta` object can be substituted with an ipfs URL to a file(s), by providing a form data field with a name matching a (`.` delimited) property path and value containing the file content (in binary string or plain text depending on file format).
```

---

Yeah it's the multipart parts that determine the _files_ that are included in the metadata not the `meta` object.